File: business/src/main/java/work/beltran/discogsbrowser/business/RxJavaSchedulers.java
package work.beltran.discogsbrowser.business;

import rx.Scheduler;

/**
 * Created by Miquel Beltran on 15.05.16.
 * More on http://beltran.work
 */
public interface RxJavaSchedulers {
    Scheduler io();

    Scheduler mainThread();
}

File: business/src/main/java/work/beltran/discogsbrowser/business/RxJavaSchedulers.kt
package work.beltran.discogsbrowser.business

import io.reactivex.Scheduler

/**
 * Created by Miquel Beltran on 15.05.16.
 * More on http://beltran.work
 */
interface RxJavaSchedulers {
    fun io(): Scheduler
    fun mainThread(): Scheduler
    fun computation(): Scheduler
}
